Improved readmes and contribution guidelines (#192)
Ogeon/palette: README.md

[![Crates.io](https://img.shields.io/crates/v/palette.svg)](https://crates.io/crates/palette/)
[![Docs.rs](https://docs.rs/palette/badge.svg)](https://docs.rs/palette)

> A color management and conversion library that focuses on maintaining correctness, flexibility and ease of use. It makes use of the type system to prevent mistakes and to support a wide range of color spaces, including user defined variants and different ways to integrate it with other libraries.

Reviewer: Reading this again, I think the second sentence could be tweaked slightly.

```suggestion
A color management and conversion library that focuses on maintaining correctness, flexibility and ease of use. It makes use of the type system to prevent mistakes, support a wide range of color spaces including user defined variants and offer different ways of integrating with other libraries.
```

[Master branch](https://ogeon.github.io/docs/palette/master/palette/index.html)

## Minimum Supported Rust Version (MSRV)

> This version has been automatically tested with version `1.48.0` and the `stable`, `beta`, and `nightly` channels. Future versions of the library may advance the minimum supported version to make use of new language features, but will consider it a breaking change.

Reviewer: ```suggestion
This version has been automatically tested with version `1.48.0` and the `stable`, `beta`, and `nightly` channels. Future versions of the library may advance the minimum supported version to make use of new language features. Advancing the MSRV will be considered a breaking change.
```
or
```suggestion
This version has been automatically tested with version `1.48.0` and the `stable`, `beta`, and `nightly` channels. Future versions of the library may advance the minimum supported version to make use of new language features, but this will be considered a breaking change.
```
